templates/style/_navbar.scss

```diff
@@ -76,14 +76,13 @@ div.nav-container {
 
     form.landing-search-form-nav {
         max-width: 1200px;
-        height: 100%;
         display: flex;
         flex-direction: row;
 
         #search-input-nav {
             display: none;
             border-left: 1px solid var(--color-border);
-            height: 100%;
+            height: calc(#{$top-navbar-height} - 1px);
```

Comment:
It's what I originally wanted to remove by using `height: 100%` and `position: relative`. Is there a "best way" between the two maybe?

Reply:
hm, yeah, maybe height: 100% is the right thing after all. Lemme go back to the drawing board on this one.

Reply:
Just to be clear: this is fine for me. It's just that I thought using `height: 100%` was the good practice. If it's not then it's completely fine. :)